inference/inference.py

```diff
@@ -173,6 +173,18 @@ def autoregressive_inference(params, ic, valid_data_full, model):
     stds = params.stds
     time_means = params.time_means
 
+    # Consider the default case to be when this function is called outside of
+    # main. In this case, we assume that the caller is train.py and therefore we
+    # do not want to log the metrics for each time-step unroll. Instead, we want
+    # to log aggegrated metrics for a *single* initial condition (ic) aggregated
+    # over time.
+    if 'is_log_time_series_metrics_to_wandb' not in params:
+      # the YParams object doesn't support params.get(k, 'default') 
+      params['is_log_time_series_metrics_to_wandb'] = False
+    
+    if 'is_log_aggregated_metrics_to_wandb' not in params:
+        params['is_log_aggregated_metrics_to_wandb'] = True  # I ran the inference code for True and False (gideond).
```

Comment:
nitpick: for things like "# I ran the inference code for True and False (gideond)." I think it's better to just leave a comment on GitHub (e.g. on the PR you've opened) versus a comment in the code which won't mean much in a matter of days/weeks.

Reply:
sounds good!